packages/graphback-cli/src/helpers/init.ts

```diff
@@ -21,7 +21,7 @@ async function installDependencies(name: string, database: string): Promise<void
   if (database === 'pg') {
     await execa('npm', ['i', '-S', 'pg'])
```

Comment:
LEt's drop redis subscriptions there - it is now added into package.json for each template